pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -13,6 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
```

Comment:
I'm assuming these changes are the result of updating generated code.  In future, please remember to separate updates to generated code into their own commits.